src/routes/Project/components/Table.js

```diff
@@ -149,14 +155,16 @@ export default class Table extends Component {
     const tableDiv = document.getElementById('table-component');
 
     let oldClip = this.props.tracks[oldTrackNum].clips[clipId];
-    let newClip = { ...oldClip };
+    let newClip = _.cloneDeep(oldClip);
```

Comment:
deep clone, where spread is just shallow clone. was running into an issue where changing newclip.starttime modified oldclip.starttime.... may have been a quirk of my chrome version but im not taking any chances